aspnetcore/signalr/client-features.md

```diff
@@ -0,0 +1,28 @@
+---
+title: SignalR client features
+author: bradygaster
+description: Learn which features are supported by the various ASP.NET Core SignalR clients.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: bradyg
+ms.custom: mvc
+ms.date: 09/18/2019
+uid: signalr/client-features
+---
+# ASP.NET Core SignalR client features
+
+## Feature distribution
+
+The table below shows the features and support for the clients that offer real-time support.
+
+| Feature | .NET Core | JavaScript | Java |
+| ---- | :-: | :-: | :-: |
+| [Server-to-client Streaming](xref:signalr/streaming)          |✔|✔|✔|
+| [Client-to-server Streaming](xref:signalr/streaming)          |✔|✔|✔|
+| Automatic Reconnection ([.NET](/aspnet/core/signalr/dotnet-client?view=aspnetcore-3.0&tabs=visual-studio#handle-lost-connection), [JavaScript](/aspnet/core/signalr/javascript-client?view=aspnetcore-3.0#reconnect-clients))          |✔|✔| |
```

Comment:
Does it make sense to include a reference to the issue tracking this work? I'd love for customers that come to this page to sound off and let us know how important it is that we close this feature gap. It would help in prioritization :)

https://github.com/aspnet/AspNetCore/issues/8711 is the issue.

Reply:
I'll move this discussion to my new PR to update this doc further.

Reply:
There is also this issue https://github.com/aspnet/AspNetCore.Docs/issues/14378 where we started listing things

Reply:
Will update my PR from that list.